packages/fast-components-react-msft/src/background/Background.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import Foundation, { HandledProps } from "@microsoft/fast-components-foundation-react";
+import {
+    DesignSystemConsumer,
+    DesignSystemProvider,
+} from "@microsoft/fast-jss-manager-react";
+import {
+    DesignSystem,
+    DesignSystemDefaults,
+} from "@microsoft/fast-components-styles-msft";
+import {
+    BackgroundHandledProps,
+    BackgroundProps,
+    BackgroundUnhandledProps,
+    DarkModeBackgrounds,
+    LightModeBackgrounds,
+} from "./Background.props";
+import { get, has, memoize } from "lodash-es";
+
+export default class Background extends Foundation<
+    BackgroundHandledProps,
+    BackgroundUnhandledProps,
+    {}
+> {
+    public static defaultProps: Partial<BackgroundProps> = {
+        tag: "div",
+        value: LightModeBackgrounds.L1,
+    };
+    protected handledProps: HandledProps<Required<BackgroundHandledProps>> = {
+        tag: void 0,
+        value: void 0,
+    };
+
+    private getDesignSystemOverrides: (color: string) => Partial<DesignSystem> = memoize(
```

Comment:
I think this should be after render

Reply:
I think that because it is being assigned a value (not simply declared) the linter sees this as a property and not a method - leaving here because it yells if I don't